packages/lesswrong/server/forumSpecificMiddleware/eaMiddleware.ts

```diff
@@ -0,0 +1,7 @@
+import type { AddMiddlewareType } from '../apolloServer';
+
+/* Add forum-specific middleware in this file. */
+
+export const eaMiddleware = (addMiddleware: AddMiddlewareType) => {
+  // app.use();
```

Comment:
I assume this is needed for WU, but not for any of the existing sites? I think I would rather remove this functionality in that case, and just have it in the WU fork

Reply:
I like this pattern. I would like to see us move in that direction for code quality and ease-of-stamping reasons.